website/content/docs/ecs/terraform/secure-configuration.mdx

```diff
@@ -16,13 +16,38 @@ A secure Consul cluster should include the following:
 
 -> **NOTE:** This page assumes that you have already configured your Consul server with the above features.
 
+## Auth Method
+
+Consul on ECS uses the [AWS IAM Auth Method](/docs/security/acl/auth-methods/aws-iam) to enable
+tasks to automatically obtain Consul ACL tokens during startup. Refer to the
+[Architecture](/docs/ecs/architecture#aws-iam-auth-method) page for an overview of the auth method
+with Consul on ECS.
+
+With the Terraform modules for Consul on ECS, the auth method is supported by default when ACLs are enabled. The ACL controller sets up the auth method on the Consul servers. The `mesh-task` module configures the ECS task definition to be compatible with the auth method.
+
+A unique task IAM role is required for each ECS task family. Since the task family represents one Consul
+service, and since the task IAM role must encode the Consul service name, task IAM roles must not be shared
+by different task families.
```

Comment:
```suggestion
A unique task IAM role is required for each ECS task family. Task IAM roles must not be shared by different task families. This is because the task family represents only one Consul service and the task IAM role must encode the Consul service name. 
```
Better? Just a nit, but we should use "since" when describing a relationship with time and "because" when describing relationships pertaining to cause/effect.